packages/fluentui/react-northstar/src/components/Tooltip/Tooltip.tsx

```diff
@@ -299,6 +311,8 @@ Tooltip.propTypes = {
     as: false,
     content: false,
   }),
+  openDelay: PropTypes.number,
```

Comment:
```suggestion
  mouseEnterDelay: PropTypes.number,
```

Let's use "mouseEnterDelay" to be consistent

Reply:
Done.